pages/common/dillo.md

```diff
@@ -0,0 +1,28 @@
+# dillo
+
+> A lightweight browser intended for slow computers.
+> More information: <https://www.dillo.org/>.
+
+- Open Dillo:
+
+`dillo`
+
+- Open Dillo with a specific window size and screen location:
+
+`dillo --geometry {{width}}x{{height}}+{{x_position}}+{{y_position}}`
+
+- Open Dillo and show a file or directory:
```

Comment:
Wait, is this a web or file browser? I think if you just write `browser` most will assume it's a web browser.

Reply:
Its a browser however just like browsers such as firefox and chrome, it open files

Reply:
So when you open a directory it opens just all files in that directory?

Reply:
Also I'd still say `web browser` in the description to avoid confusion

Reply:
Read the description.
I am also going to add another command for URLs

Reply:
> Read the description.

What do you mean? The problem is that the description alone doesn't tell me if it's a file or web browser.

Reply:
> A lightweight browser intended for slow computers.

Reply:
Should we add a file or browser before that?

Reply:
@bl-ue / @navarroaxel any opinions?

Reply:
Like `firefox` and `chromium`, I think the command description should say `web browser`.

Reply:
Ok